src/librbd/cache/rwl/LogEntry.h

```diff
@@ -90,6 +91,11 @@ class GenericWriteLogEntry : public GenericLogEntry {
   }
   void inc_map_ref() { referring_map_entries++; }
   void dec_map_ref() { referring_map_entries--; }
+  bool can_writeback();
+  virtual void copy_pmem_bl(bufferlist *out_bl) {};
+  virtual void flush(librbd::cache::ImageWritebackInterface &image_writeback,
```

Comment:
Nit: I still think this is a confusing method name vs something like `void writeback(...)`

Reply:
@dillaman renamed to `writeback(...)`